rfc9114.md

```diff
@@ -1450,17 +1450,17 @@ CANCEL_PUSH Frame {
 ~~~~~~~~~~
 {: title="CANCEL_PUSH Frame"}
 
-The CANCEL_PUSH frame carries a Push ID encoded as a variable-length integer.
-The Push ID identifies the server push that is being cancelled; see
-{{server-push}}.  If a CANCEL_PUSH frame is received that references a Push ID
+The CANCEL_PUSH frame carries a push ID encoded as a variable-length integer.
+The Push ID field identifies the server push that is being cancelled; see
+{{server-push}}.  If a CANCEL_PUSH frame is received that references a push ID
 greater than currently allowed on the connection, this MUST be treated as a
 connection error of type H3_ID_ERROR.
 
-If the client receives a CANCEL_PUSH frame, that frame might identify a Push ID
+If the client receives a CANCEL_PUSH frame, that frame might identify a push ID
 that has not yet been mentioned by a PUSH_PROMISE frame due to reordering.  If a
-server receives a CANCEL_PUSH frame for a Push ID that has not yet been
-mentioned by a PUSH_PROMISE frame, this MUST be treated as a connection error of
-type H3_ID_ERROR.
+server receives a CANCEL_PUSH frame whose Push ID has not yet been mentioned by
```

Comment:
I think that "whose" is problematic here.  And this would seem to be referring to the Push ID field value, which would be a push ID (lowercase) as used.